File: PHP/Car.php
<?php
require_once ('Account.php');

class Car {
    public $id integer;
    public $license string;
    public $driver string;
    public $passenger string;

    public function __construct($license, $driver) {
        $this->license = $license;
        $this->driver = $driver;
    }

    public function PrintDataCar(){
        echo "license: $this->license, conductor: {$this->driver->name}, document: {$this->driver->document}";
    }

}
?>

File: PYTHON/car.py
from account import Account

class Car:
    id          = int
    license     = str
    # driver      = str
    driver      = Account("", "")       # ("", "") -> obiged dato to instance the object
    passenger   = int

    def __init__(self, license, driver) -> None:
        self.license        = license
        self.driver         = driver